integration-cli/docker_cli_run_unix_test.go

```diff
@@ -1017,6 +1017,15 @@ func (s *DockerSuite) TestRunApparmorProcDirectory(c *check.C) {
 	}
 }
 
+// test that non existent apparmor profile gives helpful error
+func (s *DockerSuite) TestRunApparmorProfileNotFound(c *check.C) {
+	testRequires(c, SameHostDaemon, Apparmor)
+
+	out, _, err := dockerCmdWithError("run", "--security-opt", "apparmor=thisapparmorprofiledoesnotexistonthismachine", "busybox", "ls")
```

Comment:
Can you verify the exit code of the client here?
I believe it should be 126 since that is what is set on the container.